src/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="src",
```

Comment:
I checked the docs, and this arg is the public-facing name on PyPI (not that we're publishing it there), so it can be something more descriptive. This is not used in the `import` statement.

Reply:
makes sense. I updated it to `suggest_search_tools`. Do you think that's enough, or we should have more description?

Reply:
I think that's probably enough - we can have more description in the README. However if you want you can add the [`description` arg](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#description) in `setup()`